config/routes.rb

```diff
@@ -1,6 +1,15 @@
 Rails.application.routes.draw do
+  
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users, only: [:index, :show] do 
+    resources :posts, only: [:index, :show]
+  end
+
+  #resources :posts do
+   # resources :comments, only: [:new, :create]
+    #resources :likes, only: [:create]
+  #end
```

Comment:
**OPTIONAL** - Please remove all commented code from your project's codebase, this would help ensure that your code is clean, easy to read and maintain, please fix this 👍 👍